src/Notifaika/Discourse.hs

```diff
@@ -66,11 +66,12 @@ resultToEither :: Result a -> Either String a
 resultToEither (Success s)  = Right s
 resultToEither (Error e)    = Left e
 
-getDiscourseEvents :: (MonadIO m, MonadThrow m) => Url -> m [Event]
+getDiscourseEvents :: (SetMember Lift (Lift IO) r) => Url -> Eff r [Event]
 getDiscourseEvents baseUrl = do
     let url = baseUrl <> "/latest.json"
-    response <- liftIO $ Wreq.get url
-    jsonResponse <- asJSON response
+    jsonResponse <- lift $ do
+      response <- Wreq.get url
+      asJSON response -- XXX: because 'asJSON' require MonadThrow
```

Comment:
So, this XXX is a comment to change. And it looks strange in the current state of the code, if you don't see the previous state.

What about such comment: "asJSON doesn't actually require IO, it requires MonadThrow, but lifting it to IO should be enough in this case"?